### PR TITLE
Feature/login page

### PR DESCRIPTION
**Pull request by NguyenChiTrung1310**
Feature [Login page]:
- Add type email and password. 
- Update state and console log 
- Add router link, component 
@papercoding

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+      {/* <LoginPage /> */}
+      <Switch>
+        <Route exact path='/' component={LoginPage} />
```
Comment (papercoding) [inline src/App.tsx]: ```path``` for LoginPage should be ```/login```

File: src/App.tsx [hunk 2]
```diff
@@ -1,25 +1,20 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
-import "./styles/tailwind.output.css";
+import React from 'react';
+import './App.scss';
+import LoginPage from './pages/login/LoginPage';
+import SignupPage from './pages/signup/SignupPage';
+import './styles/tailwind.output.css';
```
Comment (papercoding) [inline src/App.tsx]: Should group imports from node_modules together and place them in the top, imports from our source code in the bottom

File: src/App.tsx [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+      {/* <LoginPage /> */}
```
Comment (papercoding) [inline src/App.tsx]: Remove the comment here when it is unused

File: src/components/Button/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Button } from 'antd';
+const ButtonComponent = ({ btnName, type }: { btnName: string; type: any }) => {
```
Comment (papercoding) [inline src/components/Button/Button.tsx]: 1. `btnName` is not a good name, should use `label` or `btnText`
2. should use `const ButtonComp: React.FC<IButton> = ({...}) => { ...}`. See `src/ExampleComponent`
3. Don't use `any`. In your editor, move mouse pointer to type in `Button` of `antd`, see the small window popup, you can know it type.

File: src/pages/login/LoginPage.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+type FieldStates = {
+  email: string;
+  password: string;
+};
+
+const LoginPage = () => {
+  const { Title } = Typography;
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    // e.preventDefault();
```
Comment (papercoding) [inline src/pages/login/LoginPage.tsx]: Remove comment when unused

File: src/pages/login/LoginPage.tsx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    const { email, password } = fields;
+
+    console.log(`Email: ${email} and password: ${password}`);
```
Comment (papercoding) [inline src/pages/login/LoginPage.tsx:23]: Remove console.log

File: src/pages/login/LoginPage.tsx [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+          ]}
+        >
+          <Input
+            prefix={<UserOutlined className='site-form-item-icon' />}
+            placeholder='Email'
+            bordered={false}
+            className='inputField emailInput'
+            onChange={(e) => {
+              const email = e.target.value;
+              setFields((prevState) => ({ ...prevState, email }));
```
Comment (papercoding) [inline src/pages/login/LoginPage.tsx]: Don't use inline function

File: src/pages/login/LoginPage.tsx [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+        >
+          <Input
+            prefix={<LockOutlined className='site-form-item-icon' />}
+            type='password'
+            placeholder='Password'
+            bordered={false}
+            className='inputField passwordInput'
+            onChange={(e) => {
+              const password = e.target.value;
+              setFields((prevState) => ({ ...prevState, password }));
```
Comment (papercoding) [inline src/pages/login/LoginPage.tsx]: Don't use inline function

File: src/App.tsx (hunk 1 above)
Comment (NguyenChiTrung1310) [reply src/App.tsx]: Fixed!

File: src/App.tsx (hunk 2 above)
Comment (NguyenChiTrung1310) [reply src/App.tsx]: Fixed!

File: src/App.tsx (hunk 3 above)
Comment (NguyenChiTrung1310) [reply src/App.tsx]: Fixed!

File: src/components/Button/Button.tsx (hunk 4 above)
Comment (NguyenChiTrung1310) [reply src/components/Button/Button.tsx]: Fixed!

File: src/pages/login/LoginPage.tsx (hunk 5 above)
Comment (NguyenChiTrung1310) [reply src/pages/login/LoginPage.tsx]: Fixed!

File: src/pages/login/LoginPage.tsx (hunk 6 above)
Comment (NguyenChiTrung1310) [reply src/pages/login/LoginPage.tsx:23]: Fixed!

File: src/pages/login/LoginPage.tsx (hunk 7 above)
Comment (NguyenChiTrung1310) [reply src/pages/login/LoginPage.tsx]: Fixed!

File: src/pages/login/LoginPage.tsx (hunk 8 above)
Comment (NguyenChiTrung1310) [reply src/pages/login/LoginPage.tsx]: Fixed!

**Review by papercoding — APPROVED** [review]

**Pull request merged**